tinyagentos/routes/cluster.py

```
@@ -125,6 +125,53 @@ async def pairing_claim(request: Request, body: PairingClaim):
+    authorises that code so the worker's poll can claim its signing key. No
+    announce or network discovery: the admin supplies the address by hand."""
+    ok, err = _require_admin(request)
+    if not ok:
+        return err
+    url = body.url.strip()
+    code = body.code.strip()
+    if not url or not code:
+        return JSONResponse({"error": "url and code are required"}, status_code=400)
+    if "://" not in url:
+        url = "http://" + url
+    parsed = urlparse(url)
```

Ruff (0.15.17)

[error] Undefined name `urlparse`

(F821)